File: src/main/java/org/model/RendezVous/rendez_vous.java
package org.model.RendezVous;

import java.time.LocalDateTime;

public class rendez_vous {
    private int id_rendez_vous;
    private int id_admin_createur;

    private String mail_user; // NEW

    private String titre;
    private String description;
    private LocalDateTime date_rendez_vous;
    private String type; // 'presentiel' | 'distance'
    private String competence;
    private LocalDateTime date_creation_rendez_vous;
    private int nombre_places;



    public rendez_vous() {}

    public rendez_vous(int id_rendez_vous, int id_admin_createur, String mail_user,
                       String titre, String description, LocalDateTime date_rendez_vous,
                       String type, String competence, LocalDateTime date_creation_rendez_vous) {
        this.id_rendez_vous = id_rendez_vous;
        this.id_admin_createur = id_admin_createur;
        this.mail_user = mail_user;
        this.titre = titre;
        this.description = description;
        this.date_rendez_vous = date_rendez_vous;
        this.type = type;
        this.competence = competence;
        this.date_creation_rendez_vous = date_creation_rendez_vous;
    }

    public int getId_rendez_vous() { return id_rendez_vous; }
    public void setId_rendez_vous(int id_rendez_vous) { this.id_rendez_vous = id_rendez_vous; }

    public int getId_admin_createur() { return id_admin_createur; }
    public void setId_admin_createur(int id_admin_createur) { this.id_admin_createur = id_admin_createur; }

    public String getMail_user() { return mail_user; }
    public void setMail_user(String mail_user) { this.mail_user = mail_user; }

    public String getTitre() { return titre; }
    public void setTitre(String titre) { this.titre = titre; }

    public String getDescription() { return description; }
    public void setDescription(String description) { this.description = description; }

    public LocalDateTime getDate_rendez_vous() { return date_rendez_vous; }
    public void setDate_rendez_vous(LocalDateTime date_rendez_vous) { this.date_rendez_vous = date_rendez_vous; }

    public String getType() { return type; }
    public void setType(String type) { this.type = type; }

    public String getCompetence() { return competence; }
    public void setCompetence(String competence) { this.competence = competence; }

    public LocalDateTime getDate_creation_rendez_vous() { return date_creation_rendez_vous; }
    public void setDate_creation_rendez_vous(LocalDateTime date_creation_rendez_vous) { this.date_creation_rendez_vous = date_creation_rendez_vous; }

    public int getNombre_places() {
        return nombre_places;
    }

    public void setNombre_places(int nombre_places) {
        this.nombre_places = nombre_places;
    }
    @Override
    public String toString() {
        return "rendez_vous{" +
                "id=" + id_rendez_vous +
                ", titre='" + titre + '\'' +
                ", date=" + date_rendez_vous +
                ", type='" + type + '\'' +
                ", competence='" + competence + '\'' +
                ", mail_user='" + mail_user + '\'' +
                '}';
    }
}

File: src_backup_phpstan/Entity/RendezVous/RendezVous.php
<?php

namespace App\Entity\RendezVous;

use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity]
#[ORM\Table(name: 'rendez_vous')]
class RendezVous
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(name: 'id_rendez_vous', type: 'integer')]
    private ?int $idRendezVous = null;

    #[ORM\Column(name: 'id_admin_createur', type: 'integer')]
    private ?int $idAdminCreateur = null;

    #[ORM\Column(name: 'mail_user', type: 'string', length: 150, nullable: true)]
    private ?string $mailUser = null;

    #[ORM\Column(name: 'titre', type: 'string', length: 200)]
    private ?string $titre = null;

    #[ORM\Column(name: 'description', type: 'text', nullable: true)]
    private ?string $description = null;

    #[ORM\Column(name: 'date_rendez_vous', type: 'datetime')]
    private ?\DateTimeInterface $dateRendezVous = null;

    #[ORM\Column(name: 'type', type: 'string', length: 50)]
    private ?string $type = null;

    #[ORM\Column(name: 'competence', type: 'string', length: 150)]
    private ?string $competence = null;

    #[ORM\Column(name: 'date_creation_rendez_vous', type: 'datetime', nullable: true)]
    private ?\DateTimeInterface $dateCreationRendezVous = null;

    #[ORM\Column(name: 'nombre_places', type: 'integer')]
    private ?int $nombrePlaces = null;
    

    public function getIdRendezVous(): ?int
    {
        return $this->idRendezVous;
    }

    public function getIdAdminCreateur(): ?int
    {
        return $this->idAdminCreateur;
    }

    public function setIdAdminCreateur(?int $idAdminCreateur): self
    {
        $this->idAdminCreateur = $idAdminCreateur;
        return $this;
    }

    public function getMailUser(): ?string
    {
        return $this->mailUser;
    }

    public function setMailUser(?string $mailUser): self
    {
        $this->mailUser = $mailUser;
        return $this;
    }

    public function getTitre(): ?string
    {
        return $this->titre;
    }

    public function setTitre(?string $titre): self
    {
        $this->titre = $titre;
        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(?string $description): self
    {
        $this->description = $description;
        return $this;
    }

    public function getDateRendezVous(): ?\DateTimeInterface
    {
        return $this->dateRendezVous;
    }

    public function setDateRendezVous(?\DateTimeInterface $dateRendezVous): self
    {
        $this->dateRendezVous = $dateRendezVous;
        return $this;
    }

    public function getType(): ?string
    {
        return $this->type;
    }

    public function setType(?string $type): self
    {
        $this->type = $type;
        return $this;
    }

    public function getCompetence(): ?string
    {
        return $this->competence;
    }

    public function setCompetence(?string $competence): self
    {
        $this->competence = $competence;
        return $this;
    }

    public function getDateCreationRendezVous(): ?\DateTimeInterface
    {
        return $this->dateCreationRendezVous;
    }

    public function setDateCreationRendezVous(?\DateTimeInterface $dateCreationRendezVous): self
    {
        $this->dateCreationRendezVous = $dateCreationRendezVous;
        return $this;
    }

    public function getNombrePlaces(): ?int
    {
        return $this->nombrePlaces;
    }

    public function setNombrePlaces(?int $nombrePlaces): self
    {
        $this->nombrePlaces = $nombrePlaces;
        return $this;
    }
}